roles/openshift_openstack/tasks/create-registry-volume.yml

```diff
@@ -0,0 +1,11 @@
+---
+- set_fact:
```

Comment:
This set_fact is unnecessary.

If you want to create vars for the sake of readability, just add a 'vars:' at the end of the task where they are used.